tsconfig.json

```diff
@@ -20,10 +20,11 @@
     // Generate separate source maps files with sourceContent included
     "sourceMap": true,
     "inlineSourceMap": false,
-    "inlineSources": true,
+    "inlineSources": false,
     // API-extractor makes use of the declarations, npm script should be cleaning these up
     "declaration": true,
-    "declarationMap": true
+    "declarationMap": true,
+    "types": []
```

Comment:
This is to avoid any types being relied upon, namely the `@types/node` I want there to always have to be an `import {Buffer} from 'buffer'` where needed, and not rely on global namespace types.